.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,36 @@
+name: "PR Checks"
+on: [pull_request]
+
+
+jobs:
+  build:
+    name: "Builds and Deploys"
+    runs-on: ubuntu-latest
+
+    steps:
+    # Check out, and set up the node/ruby infra
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+    - uses: actions/setup-ruby@v1
+
+    # Grab the handbook / playground
+    - run: git submodule init
+    - run: git submodule update
+
+    # Get all the local deps, then run danger
+    - run: yarn install
+    - run: yarn danger ci
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    # Now try build the site
+    - run: cd src/play; yarn install; yarn setup; cd ../..
+    - run: bundle install
+    - run: gulp publish
+
+    # Deploy if we can
+    - run: npx now site/ --token=$NOW_ACCESS_TOKEN --name=typescript-playbook-$PR_NUMBER
```

Comment:
Nice!

I suspect this won’t work for PRs from users outside the org (unless the secret is shared). 

But it’s great to see ZEIT Now in action! (pun intended 😉)

Reply:
Hah, I did consider this - but I figured it'd be easier to handle when it's actually on a PR without access to the keys rather than guessing now